src/tasks/installers/common-tasks.ts

```diff
@@ -94,6 +97,47 @@ export function createEclipseCheCluster(flags: any, kube: KubeHelper): Listr.Lis
   }
 }
 
+export function updateEclipseCheCluster(flags: any, kube: KubeHelper, command: Command): Listr.ListrTask {
+  return {
+    title: `Update the Custom Resource of type ${CHE_CLUSTER_CRD} in the namespace ${flags.chenamespace}`,
+    task: async (_ctx: any, task: any) => {
+      const cheOperatorCrPatchYamlPath = flags[CHE_OPERATOR_CR_PATCH_YAML_KEY]
+      let crPatch: any = {}
+      if (cheOperatorCrPatchYamlPath) {
+        if (existsSync(cheOperatorCrPatchYamlPath)) {
+          crPatch = yaml.safeLoad(readFileSync(cheOperatorCrPatchYamlPath).toString())
+        } else {
+          command.error(`Unable to find patch file defined in the flag '--${CHE_OPERATOR_CR_PATCH_YAML_KEY}'`)
```

Comment:
It is better to move this check to the beginning of server:update command.

Reply:
Yes, I did and I bit refactored CR patch file loading and usage.